spec/support/schemas/letter_base.json

```diff
@@ -1,34 +1,22 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "properties": {
-    "attributes": {
-      "properties": {
-        "name": {
-          "description": "Display name for the letter",
-          "type": "string"
-        },
-        "letter_type": {
-          "description": "Letter type used to pass as an argument to the letter download endpoint",
-          "enum": [
-            "benefits_summary",
-            "benefits_summary_dependent",
-            "benefits_verification",
-            "civil_service",
-            "commissary",
-            "proof_of_service",
-            "service_verification",
-            "medicare_part_d",
-            "minimum_essential_coverage"
-          ]
-        }
-      },
-      "type": "object"
-    },
-    "id": {
+    "name": {
```

Comment:
Should there be a `required[ "letter_type", "name" ]`?  I haven't looked at the API but I'm just assuming.

Also TIL that when using `match_response_schema` [it defaults to `strict: true`](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/schema_matchers.rb#L9) which defaults the JSON Schema Validator to assume all listed properties are required and that no unlisted properties are permitted.